File: Competitive-Progs/MaxProduct.java
package LeetCode;

import java.util.Arrays;
import java.util.HashSet;
import java.util.Set;

class MaxProduct {
    public int[] maxProduct(int[] nums) {
        return new int[] {};
    }
}

class MaxProdInArray {
    public static void main(String[] args) {
        MaxProduct aMaxProduct = new MaxProduct();
        int[] intArray = { 10, 20, 30, 40, 50 };

        Set<Integer> uniqueSet = new HashSet<Integer>();
        uniqueSet.addAll(Arrays.asList(new Integer[] { 1, 2, 3, 4, 4 }));
        Object[] result = uniqueSet.toArray();

        System.out.println("Set" + Arrays.toString(result));
    }
}

File: solns/MaxProduct.js
var x = [10, 20, 30, 40, 50];

var prod = 0;
var result = "";

for (var i = 0; i < x.length; i++) {
  for (var j = 1; j < x.length; j++) {
    if (x[i] * x[j] > prod) {
      prod = x[i] * x[j];
      result = `${i}, ${j}`;
    }
  }
}

console.log(result);
